admin-tools/data_mover.sh

```diff
@@ -0,0 +1,116 @@
+#! /bin/bash
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/5b7886792ad00a49c6ba0b6fa37ddb46f99ba820